client/stylesheets/_toCat.scss

```diff
@@ -2407,27 +2423,29 @@ i:hover {
     ul {
         margin: 0;
         padding: 0;
-        margin-top: 10px;
+        margin-top: 12px;
+        margin-left: -12px;
         display: none;
         position: absolute;
-        background-color: $color-spacegrey;
+        background-color: #404040 !important;
```

Comment:
Color literals like `#404040` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used